modules/variables/variables.tf

```diff
@@ -19,6 +19,7 @@ variable "sensitive" {
 }
 
 variable "variables" {
+  type        = any
```

Comment:
Why not `map`?

Reply:
Same as the above regarding issue #4

Reply:
Same here. Let's put a comment for clarity so it isn't reverted.

Reply:
Added 👍